documentation/docs/opal-plus/troubleshooting.mdx

```diff
@@ -0,0 +1,41 @@
+---
+sidebar_position: 10
+title: Troubleshooting
+---
+
+When something goes wrong, we are here to help.
+
+Feel free to reach out to us on Slack on our [community channel](https://bit.ly/opal-slack)
+or in your dedicated OPAL+ support channel, and we will do our best to assist you quickly.
+
+### Seeking Support
+
+If you are experiencing issues with OPAL+, please provide us with the following information:
+
```

Comment:
Full description of the issue 
Full logs from the affected OPAL server/client
For some error types we will need your OPAL configuration
A way to reproduce (e.g. a docker-compose or something) is optional and can help us reproduce and resolve this issue quicker